var/spack/repos/builtin/packages/moab/package.py

```diff
@@ -58,6 +58,7 @@ class Moab(Package):
     depends_on('mpi')
     depends_on('hdf5+mpi')
     depends_on('netcdf+mpi', when='+netcdf')
+    depends_on('metis')
```

Comment:
My conclusion was that it needs parmetis in parallel. But, when it needs parmetis, it also needs metis. I don't think there is a serial configuration that needs metis without parmetis though. Does that make sense?